digital-deck-ui/src/components/PlayersList.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div>
+    <div class="container">
+      <div
+        v-for="player in playersList"
+        :key="player"
+        class="cards"
+      >
+      <div class="info">
+       {{player.name}} <br>
+       x {{player.numOfCards}} cards
+      </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import CardDeckImageEnum from "./CardDeckImageEnum.js";
+
+export default {
+  name: "players-list",
+  computed:{
+    // filters this player
+    playersList : function(){
+      return this.$store.getters.getPlayersInfo.filter(player => player.name != this.$store.getters.getName)
```

Comment:
Maybe you can add a flag here like `return isLobby ? this.$store.getters.getPlayersInfo : this.$store.getters.getPlayersInfo.filter(player => player.name != this.$store.getters.getName)` and have `isLobby` be a prop since it makes sense to display all the players in the lobby.

Reply:
We probably also don't want to show the number of cards for each player while they're in the lobby

Reply:
Nvm, I just noticed you don't use this component in `Lobby.vue`. You probably don't have to commit this file then since Surya will be working on it